spec/factories/comments.rb

```diff
@@ -0,0 +1,28 @@
+FactoryBot.define do
+  factory :comment do
+    commenter { Faker::Name.name }
+    body { Faker::Lorem.sentence }
+
+    article
```

Comment:
My stylistic preference: use association. It makes super obvious that here we have belongs to relation. Plus, there could be cases when you need to set specific trait for this relation and you can do it with association, plus, in case if you can use method `build` instead of `create` on association it's better to build association (it won't be created in db, which means less requests to db -> faster tests)